src/applications/my-education-benefits/containers/IntroductionPage.jsx

```diff
@@ -24,73 +24,69 @@ export const IntroductionPage = ({ route, showUnverifiedUserAlert }) => {
       {showUnverifiedUserAlert && <HowToApplyPost911GiBillV2 route={route} />}
 
       <h2>Follow these steps to get started</h2>
-      <div className="process schemaform-process">
-        <ol>
-          <li className="process-step list-one">
-            <h3>Check your eligibility</h3>
-            <p>
-              Make sure you meet our eligibility requirements before you apply.
+      <va-process-list>
+        <li className="process-step list-one">
```

Comment:
I don't think you need these `className`s.

![Screenshot of the Process list code example on storybook. A `<li>` element is a top-level child of `<va-process-list>` and it doesn't have a class or className attribute.](https://user-images.githubusercontent.com/2008881/189414473-21304eff-d362-4054-92c2-92bea3a0b374.png)